gjspack/src/gjspack.js

```diff
@@ -94,7 +89,7 @@ function transform({ imported_file, resource_path, transformer }) {
 
   proc.wait(null);
   if (!proc.get_successful()) {
-    system.exit(1);
```

Comment:
This thing was infamous.
I had an old blueprint-version, which caused the transform of .blp files to fail, and this line made the entire test suite exit early, without any message or apparent reason.

Reply:
ha yes but it's tricky

when using `gjspack` as a CLI, you get the error in the terminal as expected (because it inherits blueprint stdin)

I guess we should detect if running in CLI or not, I can take care of that

Reply:
https://github.com/sonnyp/troll/issues/19